README.md

```diff
@@ -1412,21 +1411,24 @@ Table of Contents
 
 ## Vagrant Related
 
-  * [app.vagrantup.com](https://app.vagrantup.com) - HashiCorp Vagrant Cloud. Vagrant box hosting.
-  * [vagrantbox.es](https://www.vagrantbox.es/) — An alternative public box index
+  * [Vagrant Cloud](https://app.vagrantup.com) - HashiCorp Vagrant Cloud. Vagrant box hosting.
+  * [Vagrantbox.es](https://www.vagrantbox.es/) — An alternative public box index
 
 **[⬆ back to top](#table-of-contents)**
 
 ## Dev Blogging Sites
 
-  * [dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
-  * [hashnode.com](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
-  * [medium.com](https://medium.com/) — Get smarter about what matters to you.
+  * [BearBlog](https://bearblog.dev/) - Minimalist, and Markdown powered blog and website builder.
+  * [Dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
+  * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
+  * [Medium](https://medium.com/) — Get smarter about what matters to you.
 
 **[⬆ back to top](#table-of-contents)**
 
 ## Commenting Platforms
   * [Staticman](https://staticman.net/) - Staticman is a Node.js application that receives user-generated content and uploads it as data files to a GitHub and/or GitLab repository, using Pull Requests.
+  * [Giscus](https://giscus.app/) - A lightweight comments widget built arround GitHub Discussions.
+  * [Gitalk](https://gitalk.github.io/) - A little comments widget built on GitHub Issues.
```

Comment:
hmm, bit of a thin line, but these are not really SaaS, they're effectively self hosted software.  We dont tend to list JS libraries without backend services.  And the service here that's free is github in these cases.

Reply:
Ok, I'll remove it

Reply:
Should I move it to the "Other Free Resources" section?

Reply:
No, I cant list every JS library, even if I just listed ones that talk to some kind of backend it would be thousands of lines.

Reply:
Ok, I'll just remove it from the pull request then